tracing/src/macros.rs

```diff
@@ -801,6 +801,48 @@ macro_rules! event {
     );
 }
 
+/// The same as [`enabled!`], but queries subscribers specifically for an event,
+/// whereas [`enabled!`] queries for an event _or_ span.
+///
+/// See also [`span_enabled!`].
+///
+/// # Examples
+///
+/// ```rust
+/// # use tracing::{event_enabled, Level};
+/// if event_enabled!(target: "my_crate", Level::DEBUG) {
+///     // some expensive work...
+/// }
+/// ```
```

Comment:
maybe also worth including examples without an overridden target, and with field names?